lib/nxt_support/middleware/sentry_error_id.rb

```diff
@@ -9,6 +9,7 @@ def call(env)
         status, headers, body = @app.call(env)
         if status >= 500 && env['sentry.error_event_id']
           headers["Sentry-Error-Id"] = env['sentry.error_event_id']
+          headers["Access-Control-Expose-Headers"] = "*"
```

Comment:
[out of curiosity] Do we need other headers to also be accessible (why not just "Sentry-Error-Id")?

Reply:
Good question. We don't send any sensitive information in headers, so CORS isn't giving us any benefits here. I figured `*` was okay.

![image](https://user-images.githubusercontent.com/104110245/207843444-fed44d8b-f160-48b0-aa46-7e147052e813.png)

Reply:
Yes, but if we will send it someday for whatever reason maybe it's worth to enable it only for the header we need. Unlikely, but probable :) Also fits good to our "minimal changes" policy. Sorry for nitpicking ☺️

Reply:
Haha true. I guess I was lazy, because it's more code. But, on further examination, I discovered`*` probably wouldn't have worked most times, anyway:

> The value "\*" only counts as a special wildcard value for requests without credentials (requests without [HTTP cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies) or HTTP authentication information). In requests with credentials, it is treated as the literal header name "\*" without special semantics.

CORS is so annoying. Thanks.

Reply:
Yeah, I was sceptical too at first as I wasn't sure if we weren't transmitting any sensitive data. But it seems we don't, so let's leave it open. I agree %100 that CORS is annoying and good to see that I am not alone :)

Reply:
Yeah, I was sceptical too at first as I wasn't sure if we weren't transmitting any sensitive data. I agree %100 that CORS is annoying and good to see that I am not alone :)